app/src/main/res/layout/badge_list_rv_item.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
You can probably make the number of view components here a fewer number if you use constraint layout. You can try using `constraintTop_toBottomOf` to put things below others.

Reply:
There is already an issue opened to change the Layout of the list item to ConstraintLayout here #5.

Reply:
It's kinda weird to have a CR to change something when this file is being added now. If there's another one to change, why not do this "correctly" already?